lava/classes/player.py

```diff
@@ -27,6 +27,7 @@ def __init__(self, bot: "Bot", guild_id: int, node: Node):
         self._guild: Optional[Guild] = None
 
         self.autoplay: bool = False
+        self.add_song: bool = False  # This variable is to ensure that the call function is not repeated
```

Comment:
Consider using a more descriptive name for this variable such as `is_adding_song`, `is_adding_track` so we can remove the unnecessary comments while keeping the code clean and readable.